framework/db/mssql/QueryBuilder.php

```diff
@@ -71,10 +71,33 @@ public function buildLimit($limit, $offset = 0)
         }
     }
 
-//	public function resetSequence($table, $value = null)
-//	{
-//		return '';
-//	}
+    /**
+     * Creates a SQL statement for resetting the sequence value of a table's primary key.
+     * The sequence will be reset such that the primary key of the next new row inserted
+     * will have the specified value or 1.
+     * @param string $tableName the name of the table whose primary key sequence will be reset
+     * @param array|string $value the value for the primary key of the next new row inserted. If this is not set,
+     * the next new row's primary key will have a value 1.
```

Comment:
this comment does not seem to be correct. The implementation finds the maximum + 1 and not just 1.